### DISCUSSION TITLE
cannot switch to other virtual desktop

### DISCUSSION THREAD

**Discussion by IngmarBitter** [valinet/ExplorerPatcher#610] [Q&A] [answered]
The Win11 taskbar has this button: 
![image](https://user-images.githubusercontent.com/4473591/147763994-6fb7d62a-a8ab-4338-a788-acc1d09ff8cf.png)

When switching to the Win10 taskbar I get all the functionality I want, except this button is gone.  Ctrl+Win+arrow left/right is supposed to be a keyboard shortcut alternative, but that one is not working either.

How do I switch desktops when using the Win10 taskbar?

**Comment by valinet** [comment, marked as answer]
There is a similar Task View button on the Windows 10 taskbar. It can be enabled in Settings. It has a shortcut as well, look in Microsoft’s Windows 11 shortcut guide.

**Reply by IngmarBitter** [reply]
That sounds promissing.
![image](https://user-images.githubusercontent.com/4473591/147767453-34b2e492-755d-419c-ab76-1a14dd6f7083.png)
In this dialog I have "Task View" enabled, but I see no corresponding button on the taskbar.

**Reply by IngmarBitter** [reply]
And the shortcut I mentioned, ctrl+win+arrow keys is from the Win11 shortcut guide, but does not work for me. Not when in Win10 mode and also not when in Win11 mode.

**Reply by IngmarBitter** [reply]
Hmm, not sure what changed, but now it's working. Got this button now: 
![image](https://user-images.githubusercontent.com/4473591/147768068-6d1c6bf4-de5b-40ab-87f2-5c8d48a0e690.png)

**Reply by valinet** [reply]
Yeah, that’s the button.

**Reply by IngmarBitter** [reply]
Great app you made!  Thanks!
And super fast responses! Thanks!!

**Reply by valinet** [reply]
You’re welcome. Glad that it worked out.
